tensorboard/webapp/widgets/histogram/histogram_test.ts

```diff
@@ -1198,5 +1209,32 @@ describe('histogram test', () => {
         expect(onSelectTimeChangedSpy).not.toHaveBeenCalled();
       });
     });
+
+    describe('fob deselect', () => {
+      it('toggles SelectTime when in single selection', () => {
+        const fixture = createComponent('foo', [
+          buildHistogramDatum({step: 0, wallTime: 100}),
+          buildHistogramDatum({step: 5, wallTime: 400}),
+          buildHistogramDatum({step: 10, wallTime: 400}),
+        ]);
+        let onSelectTimeToggleSpy = jasmine.createSpy();
+        fixture.componentInstance.mode = HistogramMode.OFFSET;
+        fixture.componentInstance.timeProperty = TimeProperty.STEP;
+        fixture.componentInstance.linkedTime = {
+          start: {step: 5},
+          end: null,
+        };
+        fixture.componentInstance.onSelectTimeToggle = onSelectTimeToggleSpy;
+        fixture.detectChanges();
+        intersectionObserver.simulateVisibilityChange(fixture, true);
+
+        const fobComponent = fixture.debugElement.query(
+          By.directive(LinkedTimeFobComponent)
+        ).componentInstance;
+        fobComponent.removeStep.emit();
```

Comment:
"removeStep" is renamed to "fobRemoved" in #5687 Could you wait a bit and rebase this PR?